src/java.base/share/classes/java/io/Console.java

```diff
@@ -145,6 +146,124 @@ public Reader reader() {
         throw newUnsupportedOperationException();
     }
 
+    /**
+     * Writes a string representation of the specified object to this console's
+     * output stream, terminates the line and then flushes the console.
+     *
+     * <p> The effects of {@code con.println(obj)} are as if by calling
+     * {@snippet lang = java:
+     *     con.format("%s%n", obj)
+     * }
+     *
+     * @param  obj
+     *         An object whose string representation is to write.
+     *
+     * @return  This console
+     *
+     * @since 23
+     */
+    @PreviewFeature(feature = PreviewFeature.Feature.IMPLICIT_CLASSES)
+    public Console println(Object obj) {
+        throw newUnsupportedOperationException();
```

Comment:
Visually, it looks like there's a missing space; is there some other way to spell that to make it clearer its a method call?